src/Components/Publishing/Display/NewDisplayCanvas.tsx

```diff
@@ -0,0 +1,44 @@
+import { Flex } from "@artsy/palette"
+import { ErrorBoundary } from "Components/ErrorBoundary"
+import { isHTLAdEnabled } from "Components/Publishing/Ads/EnabledAd"
+import { AdDimension, AdUnit } from "Components/Publishing/Typings"
+import React, { SFC } from "react"
+import styled from "styled-components"
+
+interface DisplayCanvasProps {
+  adUnit?: AdUnit
+  adDimension?: AdDimension
```

Comment:
Are these props actually optional / will they ever not be used?

Reply:
For now I'm leaving them option bc all of the new ad work hasn't been migrated to all article types, so it was causing some test failures that @eessex was blocked by. Here's the PR for reference: https://github.com/artsy/reaction/pull/2409